packages/contracts/contracts/lib/LibAsset.sol

```diff
@@ -41,6 +56,11 @@ library LibAsset {
         IERC20(assetId).transfer(recipient, amount);
     }
 
+    /// @notice Transfers tokens from a sender to a given recipient
+    /// @param assetId Token address to transfer
+    /// @param from Address of sender/owner
+    /// @param to Address of recipient/spender
+    /// @param amount Amount to transfer from owner to spender
```

Comment:
Would have preferred to use `/* */` since that's the typical styling, but will settle for `///`

Do you prefer the latter? Just want to agree on 1 style since consistency makes things look nice :)

Reply:
Oof yeah it's mostly cus these files don't get linted and the contract docs were already done. But I can probably make these changes since it is more consistent

Reply:
I think `/** */` is actually the standard. https://tsdoc.org

Reply:
Should we not also specify the type for each param ? or is that unnecessary?

Reply:
Scratch that ^ looks like type isn't necessary to specify according to standards.

Reply:
Just realized this is solidity, so I'm assuming the /* */ format comment isn't relevant here